hail/python/hailtop/batch/backend.py

```diff
@@ -452,6 +452,26 @@ def __init__(self,
         gcs_kwargs = {'project': google_project}
         self.__fs: RouterAsyncFS = RouterAsyncFS(default_scheme='file', gcs_kwargs=gcs_kwargs)
 
+        self.regions: Optional[List[str]]
+        if isinstance(regions, str):
+            self.regions = [regions]
+        elif regions is None:
+            _regions = configuration_of('batch', 'regions', regions, None)
+            if _regions is not None:
+                self.regions = _regions.split()
+            else:
+                self.regions = None
+                warnings.warn('No default cloud regions have been specified.\n'
+                              'This may result in additional egress fees when using a Hail Batch service hosted on the '
+                              'Google Cloud Platform if the data and image do not reside in the same region in which the '
+                              'job is executed. The default behavior is to run jobs in any available region. You can '
+                              'set the default value like `hailctl config set batch/regions us-central1 us-east1` '
```

Comment:
This won't work for confusing reasons. Bash is parsing that as two separate arguments and hailctl gets confused.

This would work:
```
hailctl config set batch/regions 'us-central1 us-east1'
```
But that's a bit weird.

Maybe we should make it comma-separated? That's pretty universally expected and it plays nicely with bash.